### ISSUE TITLE
CSS auto complete popup no longer working

### ISSUE THREAD

**Issue by titoBouzout** [SublimeTextIssues/DefaultPackages#79]
_From @onetrev on May 9, 2014 18:44_

Previously, in Sublime Text 2, you could type the beginning of a class name (or ID) in your CSS file and a pop-up box with related class names would appear, as per attached image. Now in Sublime Text 3, this box can only be brought up through the "Ctrl + Space" command.

![selector-pop](https://cloud.githubusercontent.com/assets/597601/2932004/1c5da718-d7aa-11e3-903b-9942ae2c9ec9.jpg)

OS: Windows 7 SP1 & Windows 8.1 Update 1
Sublime: Build 3059

There is a thread discussing this issue here: http://www.sublimetext.com/forum/viewtopic.php?f=3&t=15037&p=57978. In my testing at least the posted temporary fix for this doesn't work on either machine I have tried it on.

*\* EDIT *\* -> actually no CSS auto complete popup is working. This is both for class names and element names, ie. "article, fieldset, etc". The only way to bring up any auto-complete window in CSS documents is with "Ctrl+Space", so clearly the auto-complete is there it's just not auto any more.

Edit by @FichteFoll: [temporary solution](https://github.com/SublimeText/Issues/issues/283#issuecomment-51718270)

_Copied from original issue: SublimeTextBugs/Issues#283_

**Comment by PierBover** [comment]
Hey @FichteFoll you said [here](https://github.com/SublimeTextIssues/Core/issues/283)

>  It's also not a bug actually. It's explicitly disabled.

Why would this be disabled?

**Comment by FichteFoll** [comment]
Probably because Jon thought that auto complete in css selectors was not needed while he was drafting the other css property completions.

**Comment by PierBover** [comment]
I hope this gets ~~fixed~~ *enhanced* soon.

Completions on CSS selectors is a crucial feature for front end devs.

**Comment by benfrain** [comment]
Agreed, from an authoring perspective we don't gain anything from disabling auto completion of selectors in CSS. Would love to see the default state amended.

**Comment by PierBover** [comment]
To add more info to this problem, it is indeed intended. It was added on [build 3019 on March 2013](http://www.sublimetext.com/3).

> CSS: Completions will not be offered when typing in selectors

**Comment by floq-design** [comment]
In case anyone finds this thread in search of a solution to this issue...

#### CSS Extended Autocompletions

Plugin for SublimeText 3 works very well:
https://packagecontrol.io/packages/CSS%20Extended%20Completions

**Just make sure you save your CSS / SCSS / LESS files to cache your classes before use.**

**Comment by Kristinita** [comment]
We wait a time, when [**patch**](https://github.com/SublimeTextIssues/Core/issues/283#issuecomment-51718270) will be added to new Sublime Text release.

Thanks.
